scipy/optimize/tests/test_direct.py

```diff
@@ -198,7 +198,7 @@ def test_epsilon(self, eps, locally_biased):
         assert result.success
 
     @pytest.mark.parametrize("locally_biased", [True, False])
-    def test_segmentation_fault(self, locally_biased):
+    def test_no_segmentation_fault(self, locally_biased):
         # test that an excessive number of function evaluations
         # does not result in segmentation fault
```

Comment:
I've renamed this test because it's name can be (and was) [misinterpreted](https://github.com/conda-forge/scipy-feedstock/pull/206#issuecomment-1195831787) to _intentionally_ create a segfault, when in fact it is testing that none occurs.